README.md

```diff
@@ -277,6 +277,46 @@ Route::group(function() {
 })->middleware('cacheResponse:600');
 ```
 
+### Tagging Routes
+
+If the cache driver you configured supports tags, you can specify a list of tags when applying the middleware:
```

Comment:
Could you specify here which drivers support tags? Or link to the relevant part in the Laravel docs?

Reply:
Done.